<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Касса</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/voicecall.png</normaloff>:/voicecall.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>64</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tabSearch">
       <attribute name="title">
        <string>Оплата</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Поиск</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="3">
            <widget class="QLineEdit" name="edtSearch"/>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="4">
            <widget class="QPushButton" name="btnSearch">
             <property name="text">
              <string>Искать</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="cmbSearchSelect">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDateEdit" name="edtRemPeriod">
             <property name="displayFormat">
              <string>dd.MM.yyyy</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTableView" name="viewAccounts"/>
          </item>
          <item>
           <widget class="QTableView" name="viewRemainders"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Подключения</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QTableView" name="viewConnections"/>
          </item>
          <item row="1" column="0">
           <widget class="QTableView" name="viewAccountPayments"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Платежи</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Оплата</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="cmbOperator">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkMulti">
             <property name="text">
              <string>Мультиплатеж</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edtSumma">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maxLength">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnPay">
             <property name="text">
              <string>Оплата</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="autoRepeat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPayments">
       <attribute name="title">
        <string>Платежи</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>64</height>
             </size>
            </property>
            <property name="title">
             <string>Фильтры платежей</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="7">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Тип договора</string>
               </property>
              </widget>
             </item>
             <item row="1" column="7">
              <widget class="QComboBox" name="cmbDogType"/>
             </item>
             <item row="1" column="3">
              <widget class="QComboBox" name="cmbPayOperator"/>
             </item>
             <item row="0" column="6">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Класс услуг</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Оператор</string>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Вид договора</string>
               </property>
              </widget>
             </item>
             <item row="0" column="8">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Дата</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QComboBox" name="cmbSection"/>
             </item>
             <item row="1" column="6">
              <widget class="QComboBox" name="cmbPayClass">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Секция</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Номер договора</string>
               </property>
              </widget>
             </item>
             <item row="1" column="8">
              <widget class="QDateEdit" name="edtPaymentsDate">
               <property name="maximumSize">
                <size>
                 <width>104</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="displayFormat">
                <string>dd.MM.yyyy</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QComboBox" name="cmbVdog">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QComboBox" name="cmbDognum">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="btnSavePayments">
               <property name="text">
                <string>Сохранить</string>
               </property>
              </widget>
             </item>
             <item row="1" column="9">
              <widget class="QPushButton" name="btnPaymentsWInfoRefresh">
               <property name="text">
                <string>Обновить</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="viewPayments"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDetail">
       <attribute name="title">
        <string>Детализация</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <widget class="QTableView" name="viewDetails"/>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>64</height>
             </size>
            </property>
            <property name="title">
             <string>Фильтр детализации</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="1" column="2">
              <widget class="QComboBox" name="cmbDetailClass">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QDateTimeEdit" name="edtDetailPeriodTo">
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>0</height>
                </size>
               </property>
               <property name="displayFormat">
                <string>dd.MM.yy HH:mm:ss</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QPushButton" name="btnDetailsRefresh">
               <property name="text">
                <string>Обновить</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Дата по</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Дата с</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="btnSaveDetails">
               <property name="text">
                <string>Сохранить</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QDateTimeEdit" name="edtDetailPeriodFrom">
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>0</height>
                </size>
               </property>
               <property name="mouseTracking">
                <bool>false</bool>
               </property>
               <property name="displayFormat">
                <string>dd.MM.yy HH:mm:ss</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Класс услуг</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabKKM">
       <attribute name="title">
        <string>ККМ</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_directories">
     <property name="title">
      <string>Справочники</string>
     </property>
    </widget>
    <addaction name="actionSettings"/>
    <addaction name="menu_directories"/>
    <addaction name="separator"/>
    <addaction name="actionLogin"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Авторизация</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>edtSearch</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>edtSumma</tabstop>
  <tabstop>btnPay</tabstop>
  <tabstop>chkMulti</tabstop>
  <tabstop>cmbOperator</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
